libs/dh/wholesale/feature-missing-measurements-log/src/lib/page.ts

```diff
@@ -0,0 +1,35 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { Component, viewChild } from '@angular/core';
+import { DhWholesaleMissingMeasurementsLogRequestLog } from './request-log';
+import { DhWholesaleMissingMeasurementsLogTable } from './table';
+
+/* eslint-disable @angular-eslint/component-class-suffix */
+@Component({
+  selector: 'dh-wholesale-missing-measurements-log-page',
+  imports: [DhWholesaleMissingMeasurementsLogRequestLog, DhWholesaleMissingMeasurementsLogTable],
+  template: `
+    <dh-wholesale-missing-measurements-log-request-log />
```

Comment:
question: have you considered instantiating the component dynamically using `WattModalService` instead of opening it by hand?

Reply:
To be honest I just "copied" another feature which looked like it, but you have a good point. I will take a look on this and change it both places :)

Reply:
The modal has now been changed to a `routed` modal like @ManBearTM did [here](https://github.com/Energinet-DataHub/greenforce-frontend/commit/2f9b85dc5176d3a82fbbe970ed406cf6acf1f39a#diff-bc05ba1f03af272e9cb4f5cd603316622354c31fb6bad491d7ca3e85c2fd18c0)